ks-svarut/src/main/java/no/difi/meldingsutveksling/ks/mapping/PropertiesHandler.java

```diff
@@ -0,0 +1,37 @@
+package no.difi.meldingsutveksling.ks.mapping;
+
+import no.difi.meldingsutveksling.config.IntegrasjonspunktProperties;
+import no.difi.meldingsutveksling.ks.Forsendelse;
+import no.difi.meldingsutveksling.ks.mapping.properties.AvgivendeSystemHandler;
+import no.difi.meldingsutveksling.ks.mapping.properties.SvarUtConfigHandler;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class PropertiesHandler implements Handler<Forsendelse.Builder> {
+    private Set<Handler<Forsendelse.Builder>> handlers;
+
+    public PropertiesHandler(IntegrasjonspunktProperties properties) {
+        this();
+        this.add(new AvgivendeSystemHandler(properties));
+        this.add(new SvarUtConfigHandler(properties));
+    }
+
+    private PropertiesHandler() {
+        this.handlers = new HashSet<>();
+    }
+
+    private PropertiesHandler add(Handler<Forsendelse.Builder> handler) {
+        handlers.add(handler);
+        return this;
+    }
+
+    @Override
+    public Forsendelse.Builder map(Forsendelse.Builder builder) {
+        for (Handler<Forsendelse.Builder> handler : handlers) {
+            builder = handler.map(builder);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Introduce a new variable instead of reusing the parameter "builder". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1226)